gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy

```diff
@@ -281,4 +281,7 @@ interface ClouddriverService {
   Response getAdhocData(@Path(value = 'groupId', encode = false) String groupId,
                         @Path(value = 'bucketId', encode = false) String bucketId,
                         @Path(value = 'objectId', encode = false) String objectId)
+
+  @GET('/appengine/storage/list')
+  List<String> getAppengineStorageAccounts()
```

Comment:
I wouldn't couple this with appengine. In the end this should just read `/storage/account/` or something similar

Reply:
Oh I see this is just for clouddriver - we'll have to fix that there later

Reply:
Yeah, let's do that now (I can do that if you want). Let's just make it a GET to `/storage`.